entity-service/src/main/kotlin/com/egm/stellio/entity/service/EntityTypeService.kt

```diff
@@ -30,6 +30,22 @@ class EntityTypeService(
         )
     }
 
+    fun getEntityTypes(contexts: List<String>): List<EntityType> =
+        neo4jRepository.getEntityTypes().map {
+            val entityType = (it["entityType"] as String)
+            EntityType(
+                id = entityType.toUri(),
+                typeName = compactTerm(entityType, contexts),
+                attributeNames = (it["properties"] as Set<String>).plus((it["relationships"] as Set<String>))
```

Comment:
should not compaction also apply to attributes?

Reply:
Indeed, fixed.